chain/near-jsonrpc-traits/src/params.rs

```diff
@@ -0,0 +1,22 @@
+use near_jsonrpc_primitives::errors::RpcParseError;
+
+use serde::de::DeserializeOwned;
+use serde_json::Value;
+
+pub trait ParamsExt<T> {
+    fn new(params: Value) -> Self;
+    fn parse(value: Value) -> Result<T, RpcParseError>
```

Comment:
I think it is better to have a default implementation here in the trait. We will not have different logic of parsing the params in both `near-jsonrpc` and ReadRPC. I also doubt anybody needs to use a different logic, but they will still be able to do so.

Do you have objections?